src/bindings/python/src/pyopenvino/graph/dimension.cpp

```diff
@@ -121,6 +121,34 @@ void regclass_graph_Dimension(py::module m) {
                 :return: Value of the dimension.
                 :rtype: int
             )");
+
+    /// Symbol-related methods: START
+    dim.def("has_symbol",
+            &ov::Dimension::has_symbol,
+            R"(
+              Check if Dimension has meaningful symbol.
```

Comment:
```suggestion
              Check if Dimension has meaningful symbol.

```
Need to add space for proper parsing by doc builder